frontend/src/components/ModalContent.vue

```diff
@@ -125,59 +122,83 @@ a {
   text-transform: none;
 }
 
-.buttons-list {
-  margin: 0.5rem 2rem;
+h1 {
+  font-family: $font-header;
+  font-size: 2.5rem;
+  color: $white;
+  word-break: break-word;
+  margin-right: auto;
+  @media only screen and (max-width: $phone) {
+    font-size: 2rem;
+    text-align: center;
+    margin-right: 0;
+  }
 }
 
-.text-list {
-  margin: 0.25rem 3rem;
+h2 {
+  font-family: $font-content;
+  font-size: 1.5rem;
+  color: black;
+  font-weight: bold;
+  word-break: break-word;
+  @media only screen and (max-width: $phone) {
+    text-align: center;
+  }
 }
 
-.buttons {
-  margin: 1rem 0.5rem 1rem 0;
+.buttons-close {
+  background-color: $blue !important;
 }
 
-.buttons--text {
+.buttons-list {
   display: flex;
-  align-content: center;
-  margin: 0.5rem;
+  flex-wrap: wrap;
+  @media only screen and (max-width: $phone) {
+    justify-content: center;
+  }
 }
 
-.content-black {
-  color: black;
-  text-align: justify;
+.buttons-chips {
+  background-color: $blue;
+  color: $white;
+  margin: 0.2rem;
 }
 
-.modal-title {
-  display: flex;
-  align-content: center;
-  font-family: $font-header;
-  font-size: 3rem;
-  padding: 2rem 0;
-  color: $white;
-  margin: 0.5rem 0 0 1.2rem;
+.buttons-text {
+  margin: 0.5rem;
 }
 
-.modal-subtitle {
-  color: black;
-  font-family: $font-content;
-  font-size: 1.5rem;
-  font-weight: bold;
-  line-height: 1.43;
-  margin: 0.5rem 0rem;
+.container-title {
+  background: $blue;
+  min-height: 5rem;
+  @media only screen and (max-width: $phone) {
+    display: flex;
+    justify-content: center;
+  }
 }
 
-.stack-list {
-  background-color: $blue;
-  color: $white;
-  margin: 0.5rem 0.5rem 0.5rem 0.5rem;
+.container-subtitle {
+  @media only screen and (max-width: $phone) {
+    display: flex;
+    justify-content: center;
+  }
 }
 
-.text-row {
-  padding: 1px;
+.partner-list {
+  display: flex;
+  align-items: center;
 }
 
-.title-header {
-  background: $blue;
+.partner-item {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  @media only screen and (max-width: $phone) {
+    padding: 2% 10%;
```

Comment:
it's not easy to predict how it will behave on different devices (resolutions), if you use rem I would stay with that, not mixed

Reply:
Good point. However if it was you to consider some kind of understand what measure would you use? Is rem the best choice?

Reply:
**Final changes:**
U used REMs both for desktop and mobile view